rero_ils/modules/loans/dumpers.py

```diff
@@ -47,14 +47,20 @@ def dump(self, record, data):
             data['patron']['name'] = ', '.join((
                 ptrn_data.last_name, ptrn_data.first_name))
 
-        # only for pending requests
+        # only for pending requests and item at desk
         if record.get('pickup_location_pid') \
-                and record.get('state') == LoanState.PENDING:
+                and record.get('state') in [
+                    LoanState.PENDING, LoanState.ITEM_AT_DESK]:
             location = Location.get_record_by_pid(
                 record.get('pickup_location_pid'))
+            library = location.get_library()
             data['pickup_location'] = {
                 'name': location.get('name'),
-                'library_name': location.get_library().get('name')
+                'library_name': library.get('name'),
+                'pickup_name': location.get(
+                    'pickup_name',
+                    f'{library["code"]}: {location["name"]}'
+                )
```

Comment:
Maybe I create a `Location.pickup_name` property method and use it here. Maybe the same behavior could be used other places into the app in future... But implementation is OK now.